xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/classes/ComputedFieldClass.java

```diff
@@ -106,23 +134,10 @@ public BaseProperty newProperty()
 
     @Override
     public void displayView(StringBuffer buffer, String name, String prefix, BaseCollection object,
-        XWikiContext context)
+            XWikiContext context)
```

Comment:
Codestyle: Looks like you added new tabulation in each method in case of line return. You might want to remove it.

Reply:
Indeed, not sure how I come up with this formatting since I has reimported and reapplied the [XWiki IntelliJ code style](https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HIntelliJIDEA4.52B), and restarted IntelliJ, but it keeps adding this tab when formatting the code...